pkg/liquidity-mining/contracts/gauges/avalanche/AvalancheRootGaugeLib.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+library AvalancheRootGaugeLib {
+    function removeDust(uint256 amount, uint256 dustModulo) internal pure returns (uint256) {
+        uint256 dust = amount % dustModulo;
+        return amount - dust;
+    }
+
+    function bytes32Recipient(address recipient) internal pure returns (bytes32) {
```

Comment:
Could also comment here why we need to cast it: some networks have different address sizes?

Reply:
Added a short description.
This is what the proxy accepts anyway, as it can be seen in the gauge code. It's just separated so that it can be tested easily (testing with the factory and the proxy outside a fork test is a nightmare).